servers/go-blocking/main.go

```diff
@@ -38,17 +46,29 @@ func main() {
 			log.Println("failed to accept connection:", err)
 			continue
 		}
+		nc := atomic.AddInt32(&numConns, 1)
+		if int(nc) >= numThreads {
+			numThreads *= 2
```

Comment:
Why `*= 2` ?
In the case of 200 concurrent connections we gonna set the GOMAXPROCS to 400 os threads, the double of c/rust.
Or am i wrong?